channel_test.go

```diff
@@ -0,0 +1,49 @@
+package gremcos
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewSafeCloseErrorChannel(t *testing.T) {
+	// GIVEN
+	// WHEN
+	safeCloseErrChan := newSafeCloseErrorChannel(1)
+
+	// THEN
+	assert.NotNil(t, safeCloseErrChan)
+}
+
+func TestMultiCloseSafeCloseErrorChannel(t *testing.T) {
+	// GIVEN
+	safeCloseErrChan := newSafeCloseErrorChannel(1)
+
+	// WHEN
+	safeCloseErrChan.Close()
+	safeCloseErrChan.Close()
+
+	// THEN
+	// there should be no panic
```

Comment:
shouldn't that be asserted with `assert.NotPanics`?

Reply:
done